docs/audits/PERSISTENCE_CACHE_AUDIT_2026-07-31.md

```
@@ -0,0 +1,150 @@
+| Model weights | IDB `vip-model-cache` | Survives (usually) | Keep; unify schema |
+| Object URLs | Video preview | Revoked / tab kill | Reconstruct from OPFS/IDB blob |
+| Forensic log | Heap array | Reload | Optional IDB or export-only |
+| Desktop↔Android | None | N/A | Portable project archive (zip of meta+blobs), user-mediated sync |
+
+---
+
+## Sync architecture proposal (desktop + Android / Iodé)
+
+**Principle:** 100% local. No cloud sync service. Cross-device continuity is **user-mediated** or **same-device multi-surface**.
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)